### PR TITLE
Set up require pools on bid test

### PR DESCRIPTION
**Pull request by canonbrother**
#### What kind of PR is this?:

/kind chore

#### What this PR does / why we need it:
Expecting new behaviour on test or may have bug

### CODE REVIEW

File: packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts [hunk 1]
```diff
@@ -217,7 +275,7 @@ async function setup (): Promise<void> {
   expect(bobColAccBefore).toStrictEqual(['8900.00000000@DFI', '545.45454546@TSLA'])
 
   aliceColAccBefore = await alice.rpc.account.getAccount(aliceColAddr)
-  expect(aliceColAccBefore).toStrictEqual(['30000.00000000@DFI', '29999.00000000@BTC', '10000.00000000@TSLA'])
+  expect(aliceColAccBefore).toStrictEqual(['29000.00000000@DFI', '29999.00000000@BTC', '10000.00000000@TSLA'])
```
Comment (canonbrother) [inline packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts:278]: before bid was 29000@DFI

File: packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts [hunk 2]
```diff
@@ -231,86 +289,181 @@ describe('placeAuctionBid success', () => {
...
+            loan: '0.44445881@TSLA'
           }
         ]
-      }
-    ])
+      })
+      const accHistory = await alice.rpc.account.listAccountHistory(aliceColAddr)
+      console.log('accHistory: ', accHistory)
+      const aliceColAcc = await alice.rpc.account.getAccount(aliceColAddr)
+      expect(aliceColAcc).toStrictEqual(['34000.00000000@DFI', '29999.50000000@BTC', '9465.00000000@TSLA'])
```
Comment (canonbrother) [inline packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts:446]: @surangap  34000@DFI vs https://github.com/DeFiCh/jellyfish/pull/854/files#r757218669

File: packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts [hunk 3]
```diff
@@ -73,11 +88,55 @@ async function setup (): Promise<void> {
   await alice.rpc.account.sendTokensToAddress({}, { [aliceColAddr]: ['10000@TSLA'] })
```
Comment (surangap) [inline packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts:88]: @chee-chyuan refer LN:85 above. it is related to https://github.com/DeFiCh/jellyfish/issues/865
We need to remove code like that from tests. and take a loan instead if required.

Comment (surangap) [reply packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts:88]: note that this only for loan tokens. tokens like `BTC` should be ok.

File: packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts [hunk 4]
```diff
@@ -446,18 +614,23 @@ describe('placeAuctionBid success', () => {
...
+      interestAmounts: [],
+      collateralValue: 55.25753134,
+      loanValue: 0,
+      interestValue: 0,
+      informativeRatio: -1,
+      collateralRatio: -1
+    })
   })
 
   // TODO(canonbrother): expected the extra will become assests (not collateral) of the vault according to pink paper
```
Comment (surangap) [inline packages/jellyfish-api-core/__tests__/category/loan/placeAuctionBid.test.ts:636]: will not be implemented. can remove this. 
pink paper shall be updated.

**Review by surangap — APPROVED** [review]

**Review by chee-chyuan — APPROVED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**